src/Samples/Net5ProjectionSample/SimpleMathProjection/SimpleMathProjection.csproj

```diff
@@ -2,12 +2,13 @@
 
   <PropertyGroup>
     <TargetFramework>net5.0-windows10.0.19041.0</TargetFramework>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <Platforms>x64</Platforms>
```

Comment:
Does this work if it is AnyCPU or no?

Reply:
It can work but I get warnings such as:

> Warning MSB3271 There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture, "AMD64", of the implementation file "..\CsWinRT\src\Samples\Net5ProjectionSample\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.dll" for "..\CsWinRT\src\Samples\Net5ProjectionSample\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.winmd". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and implementation file, or choose a winmd file with an implementation file that has a processor architecture which matches the targeted processor architecture of your project.

Reply:
ic, looks like due to the project dependency on the vcxproj.  Should we set PlatformTarget as AnyCPU?

Reply:
From offline conversation, the fact we reference the vcxproj here causes issues here as that can't target AnyCPU.  Someone will need to build the component for each architecture first and then package it in the nuspec appropriately.